packages/addons/addon-depends/jre-depends/jre-libbluray/patches/dummy

```diff
@@ -0,0 +1 @@
+this folder must exist to identify real patches folder

```

Comment:
Assuming the absolute patch path change is working, you should be able to drop the `patches/dummy` file.

Also, now that you are able to use `PKG_DEPENDS_UNPACK` you should be able to drop both of the `sources/trigger_unpack` files?

Reply:
Absolute patch dir seems to be working yes.
I still see `sources/trigger_unpack` with libXrender package. Obsolete code there too and not needed anymore?

Reply:
Yes, all obsolete, thanks for spotting those - if you can drop yours from this PR I'll clean up the others in the PR I'll be submitting for the absolute patch path.

Reply:
Yes, cleaning up and rebuilding.

Reply:
Hold fire, maybe they're not so redundant after all... checking this now. :(

Reply:
I've pushed another commit to #3661 which means those "trigger" files really are redundant now. I don't think it will have any side effects but will need to run some builds to be sure.

Reply:
Hopefully the last commit added - this change is to ensure that all the unpack functions have a consistent reference to the unpack folder (which moves about). Currently running tests, will report back later. You changes look fine btw, thanks.

Reply:
Will made my build today I hope and play with those changes.